Makefile

```diff
@@ -76,7 +76,7 @@ analyze/pull-request: $(CODE_REVIEWER_LOCATION)
 
 .PHONY: check-license
 check-license:
-	@sh check.sh || (echo run "bash check.sh" locally, commit the result and push; exit 1)
+	@bash check.sh || (echo run "bash check.sh" locally, commit the result and push; exit 1)
```

Comment:
We should use the SHELL variable that is set before or remove it entirely. I would prefer to use `SHELL:=bash` and reference it here

Reply:
I think in general it's better to use sh in Makefiles, because those scripts are simpler. SHELL is also used to invoke any command lines, so that would have potentially a big impact.
But just for this script, we *need* bash, because this script uses some advanced features.